components/binding-usage-controller/docs/proposals/usage-kind.md

```diff
@@ -37,15 +37,15 @@ UI needs two new endpoints which return:
  * all resources with a given kind. The result must be filtered out by the **metadata.ownerReference** field. If the resource contains **metadata.ownerReference**, the user should not see such a resource in the UI.
 ## Binding Usage Controller
 
-**Binding Usage Controller** takes the value of the `spec.usedBy.kind` field of the binding usage resource, looks for the corresponding usage kind which contains information about the kind of resource to be bound. The controller must handle resources even if the specified `labelsPath` field does not exist.
+Binding Usage Controller takes the value of the **spec.usedBy.kind** field of the binding usage resource, looks for the corresponding usage kind which contains information about the kind of resource to be bound. The controller must handle resources even if the specified **labelsPath** field does not exist.
 
 ## Security
 
-The administrator who adds **UsageKind** must take care of RBAC settings. BUC and ui-api-layer must be allowed to perform needed operations on resources with the kind defined or referenced in the usage kind object.
+The administrator who adds UsageKind must take care of RBAC settings. BUC and ui-api-layer must be allowed to perform needed operations on resources with the kind defined or referenced in the usage kind object.
```

Comment:
`with the kind defined or referred to in the UsageKind object.`

Reply:
with the kind defined in the UsageKind resource